src/prefabs/tabs.js

```diff
@@ -0,0 +1,1397 @@
+(() => ({
+  name: 'Tabs',
+  icon: 'TabGroupIcon',
+  category: 'CONTENT',
+  structure: [
+    {
+      name: 'Tabs',
+      options: [
+        {
+          label: 'Selected tab index',
+          key: 'defaultValue',
+          value: '0',
```

Comment:
One small thing, I think for no-coders it makes more sense if we start the active tabs from number 1.